prow/flagutil/config/config.go

```diff
@@ -43,6 +43,8 @@ type ConfigOptions struct {
 	// Inrepoconfig related flags
 	InRepoConfigCacheSize    int
 	InRepoConfigCacheDirBase string
+	// Moonraker is the centralized Inrepconfig Caching Service.
+	MoonrakerAddress string
```

Comment:
Should we document that setting this overrides the use of the IRC cache and corresponding flags?